contracts/test/modules/Offers/V1/Offers.t.sol

```diff
@@ -200,6 +205,20 @@ contract OffersV1Test is DSTest {
         vm.stopPrank();
     }
 
+    function testRevert_CannotUpdateOfferWithPreviousAmount() public {
+        vm.startPrank(address(maker));
+
+        offers.createOffer{value: 1 ether}(address(token), 0, address(0), 1 ether, 1000);
+
+        vm.warp(1 hours);
+
+        vm.expectRevert("setOfferAmount invalid _amount");
+
+        offers.setOfferAmount{value: 1 ether}(address(token), 0, 1, address(0), 1 ether);
```

Comment:
The contract requires the newly set offer amount to be different from than previous:

https://github.com/ourzora/v3/blob/60b2875a80c6cdfb13bab838b7da00c283b5b346/contracts/modules/Offers/V1/OffersV1.sol#L234-L235

Since it's handled this way, it _feels_ like a no-op in the same way we illustrate it for [an ask already existing for a token](https://github.com/ourzora/v3/blob/60b2875a80c6cdfb13bab838b7da00c283b5b346/contracts/modules/Asks/V1.1/AsksV1_1.sol#L95-L102). If that's the case, should we also reflect this behavior as such in the [associated UML diagram](https://github.com/ourzora/v3/blob/offers/uml/OffersV1/setOfferAmount.txt)?